Makefile

```diff
@@ -752,6 +755,8 @@ install.remote:
 	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
 	install ${SELINUXOPT} -m 755 $(SRCBINDIR)/podman$(BINSFX) \
 		$(DESTDIR)$(BINDIR)/podman$(BINSFX)
+	test "${GOOS}" != "windows" || \
```

Comment:
agree in principle. the reason it's like this is exit code handling and bash. A short circuit of `&&` would fail out the test in the task. Would you prefer a structured if as opposed to the inverted test condition?

Reply:
Oh, yeah. I forgot about that. Never mind!